js/reducers/homeReducer.js

```diff
@@ -15,6 +15,8 @@ function homeReducer(state = initialState, action) {
       });
     case MESSAGES_CLEAR:
       return Object.assign({}, state, { messages: [] });
+    case MESSAGES_SEARCH:
+      return Object.assign({}, state, { searchText: action.text });
```

Comment:
@linclark since #28 landed, the message reducer returns `state` as an array directly (instead of what it used to do, which was something like `{ messages: [] }`.  This patch was attaching another property into the state object for the current searchText.  Where do you think it makes sense to put this now?  Should we modify the type of the state object in the new message reducer file?